### ISSUE TITLE
Glimmer Days - Gem Series - Yes, You Can!

### ISSUE THREAD

**Issue by AndyObtiva** [planetruby/gems#67]
Hi, do you guys plan to do a GUI gem advent? If so, I got 3 libraries to share:
- https://github.com/AndyObtiva/glimmer-dsl-swt
- https://github.com/AndyObtiva/glimmer-dsl-opal
- https://github.com/AndyObtiva/glimmer-dsl-tk 

Thanks and Godspeed.

**Comment by geraldb** [comment]
Hello,  the advent series runs in december, thus, the next advent is in december 2021!

Anyways, guest post always more than welcome.   If you write up three articles for swt, opal, tk and maybe something more - I am more than happy to publish in the gem of the week series or in your own little glimmer series.  Cheers. Prosit 2021!

**Comment by AndyObtiva** [comment]
Thank you for explaining. 

I am sorry I misunderstood your mention of Advent since I saw gems post-Christmas, and thought you had changed the meaning of Advent to something more specific to Ruby that is all year long. I should have noticed you didn't use the word Advent post-December. I am a Lutheran and I still made that mistake. Shame on me.

I submitted a Pull Request:
https://github.com/planetruby/gems/pull/68

Thanks for the kind words about my blog posts. Are you suggesting I submit one every once in a while (like once a month for example) or do you think it is better for me to accumulate a number of blog posts before I submit for a series?

Cheers and Godspeed.

**Comment by geraldb** [comment]
No worries.    The comment from the merged pull request:   Wow. I try to publish your write-ups in a four day Glimmer series starting in the next days (day by day). If all works on Wednesday. The articles usually start with a short bio about the author - do you have something?
If not I try to use your short bio from the github page. Cheers. Prost.

If you want to edit or change something on the write-ups - you are more than welcome.

**Comment by AndyObtiva** [comment]
OK, I just added a very short bio on the Pull Request: https://github.com/planetruby/gems/pull/68

Godspeed.

**Comment by geraldb** [comment]
The first article in the Glimmer Days series is now published / live, see <https://planetruby.github.io/gems/glimmer/01-glimmer>   Note: I edited the article a little (mostly changing DSL to language). I am not much of a fan of acronyms :-). I hope you don't mind. Cheers.

**Comment by AndyObtiva** [comment]
Thank you.

**Comment by AndyObtiva** [comment]
I only spotted one thing that needs to be changed by the way:
```
Simple Widget Toolkit (SWT)
```
Should be:
```
Standard Widget Toolkit (SWT)
```
Thanks again. I look forward to seeing the other articles posted as well.

Cheers and Godspeed.

**Comment by geraldb** [comment]
You are welcome.  Thanks for the change I try to fix with the next update on day 2.  Cheers. Prost. PS: I posted to reddit too, see <https://old.reddit.com/r/ruby/>.

**Comment by geraldb** [comment]
The fourth and last write-up is the Glimmer Days series is now live / published, see <https://planetruby.github.io/gems/glimmer/04-glimmer-dsl-opal>   That concludes the series. I tried my best to promote Glimmer. Keep up the great work. Prosit 2021! Cheers.

**Comment by AndyObtiva** [comment]
Yes, I've been following the publishing of the articles. 

Thank you and Godspeed.

**Comment by geraldb** [comment]
Thanks again for your write-ups.  I am not a professional editor or such - I just unlinked the text a little and tried to use less acronyms - especially DSL, DSL, DSL :-) and I added a new headline / title to fit in with the series.   I hope you don't mind.   Keep up the great work (incl. the great documentation with lots of examples) .  Amazing stuff. Unfortunately, I have too much going on so I can't be any help.  Cheers. Prosit 2021!
